playwright.config.ts

```diff
@@ -3,8 +3,8 @@ import type { PlaywrightTestConfig } from '@playwright/test';
 // https://playwright.dev/docs/test-configuration
 const config: PlaywrightTestConfig = {
   testDir: './e2e',
+  timeout: 15 * 1000,
   use: {
-    headless: true,
```

Comment:
Should not this be explicitly set? Or it's the default value.

Reply:
The default value is `false`. If removed, the browser windows do not appear.